packages/yoastseo/src/scoring/renderers/AssessorPresenter.js

```diff
@@ -335,23 +333,7 @@ AssessorPresenter.prototype.removeAllMarks = function() {
  *
  * @returns {void}
  */
-AssessorPresenter.prototype.renderIndividualRatings = function() {
-	var outputTarget = document.getElementById( this.output );
-	var scores = this.getIndividualRatings();
-
-	outputTarget.innerHTML = template( {
-		scores: scores,
-		i18n: {
-			disabledMarkText: __( "Marks are disabled in current view", "wordpress-seo" ),
-			markInText: __( "Mark this result in the text", "wordpress-seo" ),
-			removeMarksInText: __( "Remove marks in the text", "wordpress-seo" ),
-		},
-		activeMarker: this._activeMarker,
-		markerButtonsDisabled: this._disableMarkerButtons,
-	} );
-
-	this.bindMarkButtons( scores );
-};
+AssessorPresenter.prototype.renderIndividualRatings = function() { };
```

Comment:
Is this okay to have a noop function for `renderIndividualRatings`? I wonder if there is a way to check the specific function that used to use this function to make sure that we don't break anything?